### PR TITLE
Seelie Balancing, Addition of Seelie Hand and Servant

### PR DESCRIPTION
**Pull request by Simply-Sydney**
## About The Pull Request

Short synopsis:
Altered global define for Seelie, and two new lists to help include them in jobs such as Hand, Servant, and Seamster. Allowed Seelie to become hand and servant, with custom flavor text on starting, as well as custom flavor text for starting as a prisoner. Reworked water bolt spell, rebalance of cooldown delay on most Seelie spells New additions of spells, such as the AOE slowdown spell for Seelie Hand and the Sate Crop spell for Seelie Maid.

Explanation:
Seelie spells felt extremely long in cooldown, to the point almost of uselessness. Especially for things like Tame Rous, which sometimes did not work correctly even if all conditions were met. New cooldowns are as follows:
* summon_rat           - 2 Minutes -> 90 Seconds
* strip (jester only)    - 6 Minutes -> 2 Minutes
* animate_object      - 8 Minutes -> 5 Minutes
* roustame                - 3 Minutes -> 1 Minute
* seelie_kiss               - 4 Minutes -> 90 Seconds (rarely used, felt unimpactful)
* splash (water bolt)  - 3 Minutes -> 35 Seconds (this low to support maid cleaning)
* summon_berry       - 3 Minutes -> 2 Minutes

Furthermore, animate object was altered to cost slightly less fatigue, it used to take roughly 1/3 of the bar, now slightly less.
Seelie Kiss was also updated to go from injecting 1 unit of red and blue pot to 2
Also, a new spell was added to support Seelie Maids: Sate Crop
This will satisfy the nutrients for crops around the seelie similar to the Dendor spell - but it will NOT bless them, only sate
Water Bolt was altered to affect a 3x3 area, and will now clean floors as well as water crops, also to help with Seelie maids

Finally, custom tutorial text was added for Seelie only joining as Hand, Seamster, Servant, or Prisoner roles. 
Now they will say something more custom tailored to Seelie when joining the round.

Supporting pictures and proof of testing:

image
^ Sate Crop and Water Bolt

image
^ Custom tutorial text example, plus message for the spell given to Seelie Hand

image
^ Seelie Hand spell will immobilize for 3 SECONDS, not long, but maybe long enough to stop fighting in court

Screenshot 2024-11-01 175000
Screenshot 2024-11-01 175129
^ Water bolt cleaning effect before and after

Screenshot 2024-11-01 183822
^All playable roles for Seelie

## Why It's Good For The Game

Seelie spells were in a rough spot, I wasnt sure how long to originally make the cooldowns and way overestimated how impactful they would be. This is an attempt to alleviate that after a lot of feedback.

New roles for Seelie will hopefully make them feel more integrated into the town, without the roles being unfitting for their design philosophy!

### CODE REVIEW

**Comment by Nakalon** [comment]
No i hate seelies clsing this Pr aaaaaaaaaaaaaaaaaaaaa

**Comment by Nakalon** [comment]
I love it, love you, love seelie, question. How many spells does each class have?

I take hand has 3, town seelie has like 6, maid has 2 right? Seamster having 1.

Can you perhaps give something to seamster? I don't want her to cry

**Comment by Simply-Sydney** [comment]
> I love it, love you, love seelie, question. How many spells does each class have?
> 
> I take hand has 3, town seelie has like 6, maid has 2 right? Seamster having 1.
> 
> Can you perhaps give something to seamster? I don't want her to cry

Thats correct yes! Unless Wow5000's PR to give Town Seelie the Force Push spell got merged

**Comment by Simply-Sydney** [comment]
Oh wait Seelie Servant has 5, two of which are for making and taming rous for butchering and getting small amounts of milk / meat

**Comment by Gyrandola** [comment]
They are stealing our jobs

**Comment by TheFenix323** [comment]
Not speed merged yet? smh

**Comment by Simply-Sydney** [comment]
> Not speed merged yet? smh

The hell do I bribe these guys for? Smh my head...

**Comment by Nakalon** [comment]
> > Not speed merged yet? smh
> 
> The hell do I bribe these guys for? Smh my head...

I would but I want everyone elses opinion first because im biassed for seelies

**Comment by Gyrandola** [comment]
Im not sure whos gonna respect a seelie hand, but that's a IC issue I suppose

File: code/modules/jobs/job_types/roguetown/nobility/hand.dm [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
 	spawn_positions = 1
-	allowed_races = RACES_SHUNNED_UP
+	allowed_races = RACES_SHUNNED_UP_PLUS_SEELIE
 	allowed_sexes = list(MALE, FEMALE)
 	outfit = /datum/outfit/job/roguetown/hand
 	display_order = JDO_HAND
 	tutorial = "You owe everything to your liege. Once, you were just a humble friend- now you are one of the most important men within the realm itself. You have played spymaster and confidant to the Noble-Family for so long that you are a vault of intrigue, something you exploit with potent conviction. Let no man ever forget whose ear you whisper into. You've killed more men with those lips than any blademaster could ever claim to."
 	whitelist_req = TRUE
 	give_bank_account = 44
-	min_pq = 3
```
Comment (Nakalon) [inline code/modules/jobs/job_types/roguetown/nobility/hand.dm:15]: why did you lower PQ for hand

Comment (Simply-Sydney) [reply code/modules/jobs/job_types/roguetown/nobility/hand.dm]: Fixed!

**Comment by Nakalon** [comment]
Merging this because its lovely, Mori will get me demoted

**Pull request merged**